scrinium.csl

```diff
@@ -0,0 +1,794 @@
+          </else>
+        </choose>
+        <text macro="event" prefix=" "/>
+        <text variable="event-place" prefix=", "/>
+        <text macro="issued" prefix=", "/>
+      </else-if>
+      <else-if variable="publisher-place publisher" match="any">
+        <group prefix=". " delimiter=", ">
+          <choose>
+            <if type="thesis">
+              <text variable="genre" text-case="capitalize-first"/>
+            </if>
+          </choose>
+          <text macro="publisher"/>
+          <text macro="issued"/>
+        </group>
+      </else-if>
+      <else>
+        <text macro="issued" prefix=", "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="access-note">
+    <group delimiter=", ">
+      <choose>
+        <if type="graphic report" match="any">
+          <text macro="archive-note"/>
+        </if>
+        <else-if type="article-journal article-magazine article-newspaper bill book chapter graphic legal_case legislation motion_picture paper-conference report song thesis" match="none">
+          <text macro="archive-note"/>
+        </else-if>
+      </choose>
+      <choose>
+        <if variable="URL">
+          <group delimiter=". ">
+            <choose>
+              <if variable="page" match="none">
+                <text value="n.p."/>
+              </if>
+            </choose>
+            <group delimiter=" " prefix="[" suffix="]">
+              <text term="cited"/>
+              <date variable="accessed" form="text"/>
+            </group>
+            <group delimiter=": ">
+              <text term="online" text-case="capitalize-first"/>
+              <text variable="URL"/>
+            </group>
+          </group>
+        </if>
+      </choose>
+    </group>
+  </macro>
+  <macro name="access">
+    <group delimiter=". ">
+      <choose>
+        <if type="graphic report" match="any">
+          <text macro="archive"/>
+        </if>
+        <else-if type="article-journal article-magazine article-newspaper bill book chapter graphic legal_case legislation motion_picture paper-conference report song thesis" match="none">
+          <text macro="archive"/>
+        </else-if>
+      </choose>
+      <choose>
+        <if variable="URL">
+          <choose>
+            <if variable="page" match="none">
+              <text value="No pages"/>
+            </if>
+          </choose>
+          <group delimiter=" ">
+            <text term="cited" text-case="capitalize-first"/>
+            <date variable="accessed" form="text"/>
+          </group>
+          <group delimiter=": ">
+            <text term="online" text-case="capitalize-first"/>
+            <text variable="URL"/>
+          </group>
+        </if>
+      </choose>
+    </group>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-names="true">
+    <layout suffix="." delimiter="; ">
+      <choose>
+        <if position="ibid-with-locator">
```

Comment:
if you just want to get rid of ibid, delet lines 733 to 746 and then change the beginning of line 747 and 753 to `<if>` and `</if>` respectively.